### PR TITLE
enhancements/payments-history

### PR DESCRIPTION
**Pull request by koslib**

### CODE REVIEW

File: tests/test_payments_client.py [hunk 1]
```diff
@@ -14,7 +15,7 @@
 class PaymentsClientTests(CheckoutSdkTestCase):
     def setUp(self):
         super().setUp()
-        self.http_client = HttpClient(Config())
+        self.http_client = HttpClient(Config(secret_key="sk_test_4576e25e-1e16-49a7-8f85-e91c3291fc09"))
```
Comment (dreamPathsProjekt) [inline tests/test_payments_client.py]: secret_key has to be removed here. It is used to test things locally.

File: checkout_sdk/payments/payments_client.py [hunk 2]
```diff
@@ -87,6 +95,13 @@ def get(self, id):
 
         return PaymentProcessed(self._send_http_request('charges/{}'.format(id), HttpMethod.GET))
 
+    def history(self, id):
+        self._log_info('Getting charge history for {}'.format(id))
```
Comment (dreamPathsProjekt) [inline checkout_sdk/payments/payments_client.py:99]: Review this if compatibility with Python 2.7 is needed.

File: checkout_sdk/common/charge.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+class Charge:
```
Comment (dreamPathsProjekt) [inline checkout_sdk/common/charge.py:1]: Review this if compatibility with Python 2.7 is needed.

**Review by dreamPathsProjekt — CHANGES_REQUESTED** [review]
Please see individual minor comments. Change only needed on HttpClient() initialization on test_payments_client.py to remove secret_key for other users to use their own.

File: checkout_sdk/payments/payments_client.py (hunk 2 above)
Comment (koslib) [reply checkout_sdk/payments/payments_client.py:99]: No, the package supports Python 3.4+

File: tests/test_payments_client.py (hunk 1 above)
Comment (koslib) [reply tests/test_payments_client.py]: Absolutely correct, thanks for catching this @dreamPathsProjekt!

**Pull request closed without merging**